src/ts/cmd/endpoint-registry-ui/i40-aas-registry-ui/webapp/controller/CreateDescriptor.controller.js

```diff
@@ -0,0 +1,313 @@
+
+
+sap.ui.define([
+  "sap/ui/core/mvc/Controller",
+  "sap/ui/core/routing/History",
+  "sap/ui/model/json/JSONModel",
+  "sap/m/MessageToast",
+  "sap/m/MessageBox",
+
+
+], function (Controller, History, JSONModel, MessageToast, MsgBox) {
+  "use strict";
+
+  return Controller.extend("i40-aas-registry-ui.i40-aas-registry-ui.controller.CreateDescriptor", {
+
+    onInit: function () {
+
+      this.initiateModel();
+    },
+
+    getInputs: function () {
+      return {
+        inputAasId: this.getView().byId("InputAasId"),
```

Comment:
```suggestion
        inputAasId: this.byId("InputAasId"),
```
A bit cleaner way would be the one above